ui/src/notifications/Notifications.tsx

```diff
@@ -126,57 +129,41 @@ export default function Notifications({
           </div>
         </header>
       )}
-      <section className="h-full w-full overflow-y-scroll bg-gray-50">
+      <section className="flex h-full w-full flex-col space-y-6 overflow-y-scroll bg-gray-50 p-6">
         <Helmet>
-          <title>
-            {group
-              ? `All Notifications for ${group?.meta?.title} ${title}`
-              : title}
-          </title>
+          <title>{group ? `${group?.meta?.title} ${title}` : title}</title>
```

Comment:
Maybe add some fallback value if `meta` is undefined to avoid displaying `undefined`? Or it's impossible that meta is not there?